examples/policy/config_label_config.yaml

```diff
@@ -0,0 +1 @@
+policy: example_policy.config_label_policy
```

Comment:
How about have one such func that's the composition of the 2 funcs?

Also, can we add & test a func that rejects certain tasks, e.g., where image_id is not a docker image?

We can also test rejecting "autostop / autodown not set", but that isn't captured in the current interface yet. Shall we briefly revisit the design?

Reply:
Just added a few examples. As we discussed offline, let's keep a single function for now for simplicity.